client/client.go

```diff
@@ -1905,6 +1949,104 @@ func (c *client) WatchGlobalConfig(ctx context.Context, configPath string, revis
 	return globalConfigWatcherCh, err
 }
 
+func (c *client) Put(ctx context.Context, key, value []byte, opts ...OpOption) (*pdpb.PutResponse, error) {
+	options := &Op{}
+	for _, opt := range opts {
+		opt(options)
+	}
+
+	if span := opentracing.SpanFromContext(ctx); span != nil {
```

Comment:
how about exract the trace code as one funtion?

Reply:
Will do it in another PR